### DISCUSSION TITLE
PrimeNG Calendar does not allow to select max year

### DISCUSSION THREAD

**Discussion by mittal-shah** [primefaces/community#322] [PrimeNG] [answered]
PrimeNG Calendar does not allow to select max year. To reproduce set max-date to "2005-05-01", Open date picker, select current year link 2023, click previous button until you see 2005. Notice, that 2004 is available to pick but not 2005. Any solutions or workaround? Should this be turned to an issue?

![image](https://github.com/primefaces/community/assets/27385176/0a1b84ee-c663-45f6-ba1e-bc2f0fce2a8f)

See this demo to verify https://stackblitz.com/edit/xku2nb?file=src%2Fapp%2Fdemo%2Fcalendar-minmax-demo.ts

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for sharing the issue with us could you please create a bug report on the PrimeNG repo and share a working stackblitz example so we can include it in future milestones.

Stackblitz link you've shared is broken btw.

**Reply by mittal-shah** [reply]
https://github.com/primefaces/community/issues/330 Issue created. Thanks
